variables.tf

```
@@ -111,3 +100,30 @@ variable "host_name" {
   default     = "bastion"
   description = "The Bastion hostname created in Route53"
 }
+
+variable "user_data_template" {
+  type        = string
+  default     = "userdata/amazon-linux.sh"
+  description = "User Data template to use for provisioning EC2 Bastion Host"
+}
+
+variable "ami_filter" {
+  description = "List of maps used to create the AMI filter for the action runner AMI."
+  type        = map(list(string))
+
+  default = {
+    name = ["amzn2-ami-hvm-2.*-x86_64-ebs"]
+  }
+}
+
+variable "ami_owners" {
+  description = "The list of owners used to select the AMI of action runner instances."
+  type        = list(string)
+  default     = ["amazon"]
+}
+
+variable "enable_ssm" {
```

See: https://docs.cloudposse.com/reference/best-practices/terraform-best-practices/#use-feature-flags-to-enabledisable-functionality

```suggestion
variable "ssm_enabled" {
```

---

Updated